GVRf/Sample/gvr-cubemap/src/org/gearvrf/cubemap/CubemapScript.java

```diff
@@ -62,6 +65,26 @@ public void onInit(GVRContext gvrContext) {
         GVRMaterial cubemapMaterial = new GVRMaterial(gvrContext,
                 GVRMaterial.GVRShaderType.Cubemap.ID);
         cubemapMaterial.setMainTexture(futureCubemapTexture);
+        
+        ArrayList<Future<GVRTexture>> futureTextureList = new ArrayList<Future<GVRTexture>>(6);
+        futureTextureList.add(gvrContext
+                .loadFutureTexture(new GVRAndroidResource(gvrContext,
+                        R.drawable.back)));
```

Comment:
you appear to be missing 'back', 'right', 'front', 'left', 'top', 'bottom' from your drawables directory.  they appear to not be checked-in.